langchain/llms/__init__.py

```diff
@@ -24,10 +32,14 @@
 type_to_cls_dict: Dict[str, Type[BaseLLM]] = {
     "ai21": AI21,
     "anthropic": Anthropic,
+    "cerebriumai": "CerebriumAI",
     "cohere": Cohere,
+    "forefrontai": "ForefrontAI",
+    "gooseai": "GooseAI",
     "huggingface_hub": HuggingFaceHub,
     "nlpcloud": NLPCloud,
     "openai": OpenAI,
+    "petals": "Petals",
```

Comment:
This looks like the cause of the failing lint test. `"gooseai": "GooseAI"` should be `"gooseai": GooseAI`

Error:
https://github.com/hwchase17/langchain/actions/runs/4146938778/jobs/7173220060#step:6:20

![image](https://user-images.githubusercontent.com/29989939/218428380-3a117ccd-7b64-47e5-bdea-b011b3c6089c.png)

Reply:
Ok.Thank you for bringing attention to this. I am unfamiliar with Lint. I will resolve this.

Reply:
@batmanscode It looks like everything is passing now. Thank you again.

Reply:
Nice, glad it worked!